### PR TITLE
Made lastName Last Name in student registration

### PR DESCRIPTION
**Pull request by rroloff**
Fixes #[issue-number]

### CODE REVIEW

**Comment by rroloff** [comment]
this seems to be merged to master already

File: screens/MatchesScreen.js [hunk 1]
```diff
@@ -138,6 +139,20 @@ export class MatchesScreen extends Component {
       </Container>
     );
   }
+
```
Comment (kmui2) [inline screens/MatchesScreen.js]: @rroloff, remove these exact changes in MatchesScreen.js.

File: .circleci/config.yml [hunk 2]
```diff
@@ -44,7 +44,18 @@ jobs:
           command: cat ./coverage/lcov.info | ./node_modules/.bin/coveralls
 
       - add_ssh_keys
-      - run: ssh-keyscan 104.248.57.156 >> ~/.ssh/known_hosts
-      - run:
-          name: Deploy to Development Server
-          command: npm run deploy
+      - deploy:
```
Comment (kmui2) [inline .circleci/config.yml]: Also remove these changes as well.

**Comment by kmui2** [comment]
fixed in master

**Pull request closed without merging**